packages/tools/examples/planarFreehandContourSegmentationTool/index.ts

```diff
@@ -340,6 +264,40 @@ addButtonToToolbar({
   },
 });
 
+let smoothingEnabled = false;
+const toggleSmoothingButtonContainer = document.createElement('span');
+
+// Reserve some space in the toolbar because this input is added later
+toolbar.appendChild(toggleSmoothingButtonContainer);
+
+function addToggleSmoothingButton(toolGroup) {
```

Comment:
We have a toggle in utils, use that, it is the same as show/hide all segments element